yaml/pois.yaml

```diff
@@ -414,3 +414,45 @@ filters:
     min_zoom: 15
     output:
       kind: {col: tags->whitewater}
+  - filter: {aerialway: pylon}
```

Comment:
Good question! We have existing features like traffic lights, waste bins, & etc in the `pois` layer now (for historic reasons). Would those also move into a `utilities` layer? Maybe it should be called something more generic like `infrastructure` instead (and also move the aerialway=pylon)? Would barrier lines move from the existing `boundaries` layer to a new `infrastructure` layer? Would most amenity and shop values go into the `pois` layer, but many of the man_made values go into `infrastructure` instead?

Moving features into a dedicated `infrastructure` layer would mean there's less chance of them getting displayed mixed in with regular business `pois`, but it also increases the complexity of where to find things to build the map style. We had been discussing centralizing even more "poi" like features into the `pois` layer – we do this now for shops and other tags present on buildings, we peal those off and put them into the `pois` layer leaving only (icon-less) label placements behind. But in the `landuse` layer label placements should generally have icons so we already mix this up a bit. 

Where would features like `amenity: bbq` and `amenity: life_ring` and `amenity: picnic_table` go? Seems like most of these features could just go into an `infrastructure` layer just as well as a `pois` layer. Sure they have an amenity tag from OSM, but looking at it not knowing the history it's not clear. Less layers seems better.